src/frontend/src/components/toast.ts

```diff
@@ -0,0 +1,55 @@
+import { html, render } from "lit-html";
+import { irregularity } from "./irregularity";
+
+export const removeToast = (toast: HTMLElement): void => {
+  toast.addEventListener("animationend", () => {
+    toast.remove();
+  });
+  toast.classList.add("c-toast--closing");
+};
+
+const createToast = (messageStr: string): HTMLDivElement => {
+  const message = html`${messageStr}`;
+  const $toast = document.createElement("div");
+  $toast.classList.add("c-toast");
+  render(
+    irregularity({
+      message,
+      closeFn: () => removeToast($toast),
+    }),
+    $toast
+  );
+  return $toast;
+};
+
+/**
+ * Get the container for the toasts, or create it if it doesn't exist.
+ * @returns the container reference for the toasts
+ */
+const getOrCreateToaster = (): Element | null => {
+  const $toaster = document.querySelector("[data-toaster]");
+  if (!$toaster) {
+    const $toaster = document.createElement("div");
+    $toaster.classList.add("c-toasts");
+    const $container = document.createElement("div");
+    $container.classList.add("l-container");
+    $container.setAttribute("data-toasts", "");
+    $toaster.setAttribute("data-toaster", "");
+    $toaster.appendChild($container);
+    document.body.appendChild($toaster);
```

Comment:
What happens if other children are appended after the toaster? will the toasts still be displayed correctly?